app/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "opg-modernising-lpa",
+  "version": "0.0.1",
+  "scripts": {
+    "build": "yarn build:js && yarn build:css && yarn build:images && yarn build:fonts",
+    "build:js": "esbuild --bundle web/assets/main.js --minify --outdir=web/static/javascript --sourcemap --resolve-extensions=.mjs,.js",
+    "build:css": "sass --load-path . --style compressed web/assets/main.scss web/static/stylesheets/all.css",
+    "build:images": "mkdir -p web/static/assets/images && cp node_modules/govuk-frontend/govuk/assets/images/* node_modules/@ministryofjustice/frontend/moj/assets/images/* web/static/assets/images",
+    "build:fonts": "mkdir -p web/static/assets/fonts && cp node_modules/govuk-frontend/govuk/assets/fonts/* web/static/assets/fonts",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "@ministryofjustice/frontend": "^1.4.2",
+    "dart-sass": "^1.25.0",
+    "govuk-frontend": "^4.2.0",
+    "jquery": "^3.6.0"
```

Comment:
We could probably drop the jquery dependency and make the build even simpler if we just got the compiled JS from GDS like here https://github.com/mattmachell/esbuild-boilerplate/blob/main/example/main.js#L1 what do you think?

Reply:
I shuddered at having to install it but if we want to rely on any components in the MOJ design system it looks like we need it - https://design-patterns.service.justice.gov.uk/get-started/setting-up-javascript/ (unless there's a similar workaround for MOJDS as well?)

Reply:
Leave it in for now then and we can always revisit it as we start to build more of the journey etc, all looking good mate!